### DISCUSSION TITLE
Educative Scraper v4.9 [07/07/2022]

### DISCUSSION THREAD

**Discussion by anilabhadatta** [anilabhadatta/educative.io_scraper#10] [General]
- version 4.9
- Fixed multiple bugs related to selectors as Educative changed their DOM.
- Added a few optimizations.

- If you want to Run/Modify the script manually by git clone then download the Chrome-bin supported by the OS from below.
- Win Chrome Version 96
- Mac Chrome Version 98
- Linux Chrome Version 98

**Full Changelog**: https://github.com/anilabhadatta/educative.io_scraper/compare/Educative-Scraper-4.7.3...Educative-Scraper-v4.9

<hr /><em>This discussion was created from the release <a href='https://github.com/anilabhadatta/educative.io_scraper/releases/tag/Educative-Scraper-v4.9'>Educative Viewer v4.9 [07/07/2022]</a>.</em>

**Comment by sarveshkhanna** [comment]
how do i download the chrome bin?
